avidemux_fr.ts

```diff
@@ -711,7 +711,7 @@ Ignorer les informations de timing ?</translation>
     <message>
         <location line="-912"/>
         <source>Play [SPACE]</source>
-        <translation>Lire [SPACE]</translation>
+        <translation>Lire [ESPACE]</translation>
```

Comment:
Please be aware of this translation having no effect, it is always overridden. Shortcuts in square brackets are not yet translatable (please have a look at `MainWindow::widgetsUpdateTooltips` and especially the `getActionShortcutString` function).